packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -458,25 +519,91 @@ class AppSizeController {
     diffMap['n'] = isDeferredApp.value ? _entireAppNodeName : _rootNodeName;
 
     // TODO(peterdjlee): Try to move the non-active tree generation to separate isolates.
-    _combinedDiffTreeRoot = generateDiffTree(
-      diffMap,
-      DiffTreeType.combined,
-      skipNodesWithNoByteSizeChange: !isDeferredApp.value,
+    // Entire app or root (for non-deferred):
+    final diffTreeMap = _generateDiffTrees(diffMap);
```

Comment:
Let's actually set the `diffTreeMap` (and the `mainDiffTreeMap`, etc) as a top-level variable. So instead of:
```
    _combinedDiffTreeRoot
    _increasedDiffTreeRoot
    _decreasedDiffTreeRoot
  _mainCombinedDiffTreeRoot
   etc..
``` 
We only have:

```
_diffTreeMap
_mainDiffTreeMap
_deferredDiffTreeMap
```